### ISSUE TITLE
If you open gimp from "Open with another application" context menu, it will not have a dark border

### ISSUE THREAD

**Issue by skoruppa** [hardpixel/dark-variant#4]
**Describe the bug**
If you open gimp from "Open with" context menu, it will not have a dark border. Opening it from applications menu works fine

**To Reproduce**
1. Configure extension to apply dark theme for gimp
2. Click with right mouse button on an image file in nautilus
3. Select *Open with another application*
4. Select Gimp and confirm

**Expected behavior**

**Screenshots**
Open With menu:
![image](https://user-images.githubusercontent.com/899429/191896687-fc7fef66-d7e1-49b7-a707-7e6269f9c060.png)
Gimp with light border because it was opened through "**Open with another application**":
![image](https://user-images.githubusercontent.com/899429/191896728-faa14af0-af9f-4916-8e78-05bacd8b344a.png)
Gimp opened through applications menu
![image](https://user-images.githubusercontent.com/899429/191896864-37292be0-2b25-4b56-817b-e5fc0145906a.png)

**Environment (please complete the following information):**
 - OS: Ubuntu 22.04
 - GNOME Shell 42
 - Dark Variant 8

**Comment by jonian** [comment]
Hi @skoruppa, thanks for the report. Can't replicate this on arch linux. Looks like an ubuntu issue. Is gimp install through Snap/Flatpak?

**Comment by skoruppa** [comment]
No, unfortunately not :( Official package from Ubuntu in this case is just a [regular deb](https://packages.ubuntu.com/jammy/gimp)
